documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot-nosql.adoc

```diff
@@ -83,6 +84,13 @@ The array lists regular expressions which match {data-collection}s by their full
 |An optional `type` component of the `data` field of a signal that specifies the type of snapshot operation to run. +
 Currently supports the `incremental` and `blocking` types. +
 If you do not specify a value, the connector runs an incremental snapshot.
+
+|6
+|`additional-conditions`
+| An optional array that specifies a set of additional conditions that the connector evaluates to determine the subset of records to include in a snapshot. +
+Each additional condition is an object with `data-collection` and `filter` properties.
+You can specify different filters for each data collection. +
+* The `data-collection` property is the fully-qualified name of the data collection for which the filter will be applied.
```

Comment:
It seems that something is missing here. The asterisk (`*`) is supposed to represent a bullet in an unordered list.  I think that this text is supposed to mirror the text in the description for `additional-conditions` that's in the file `proc-triggering-an-incremental-snapshot-kafka.adoc` (L36-44). Is that right?
That description includes a definition list with two entries, `data-collection` and `filter`. 

To modify the text in this PR to reflect that definition, it's necessary to add the `a` style operator to the specifier for the description column in the table's attributes list on L56 (or place the operator just in front of the cell separator on L90  (i.e., `a|`). 
With that change in place, we can then replace the current text in the table cell with a definition list, by adding the following:

```suggestion
For each additional condition object you can set the following parameters:

data-collection::
The fully-qualified name of the collection that the filter applies to. 
You can apply different filters to each collection. 

filter:: Specifies column values that must be present in a database record for the snapshot to include it, for example, `"color='blue'"`.
```

Reply:
> I think that this text is supposed to mirror the text in the description for additional-conditions that's in the file proc-triggering-an-incremental-snapshot-kafka.adoc (L36-44). Is that right?

I think it shouldn't match the kafka adoc. It should rather match the `additional-conditions` of `proc-triggering-an-incremental-snapshot.adoc` (L127-132).